janusgraph-examples/example-remotegraph/README.md

```diff
@@ -0,0 +1,75 @@
+# Remote Graph
+
+## About Remote Graph
+
+[Gremlin Server](http://tinkerpop.apache.org/docs/3.2.6/reference/#gremlin-server)
+allows a JanusGraph instance to run on a standalone server. This enables
+multiple clients to connect to the same JanusGraph instance. It also allows
+clients to connect via non-JVM languages. By default, the Gremlin Server
+communicates over WebSocket.
+
+A Java program can connect to the Gremlin Server in two ways:
+* [Gremlin Driver](http://tinkerpop.apache.org/docs/3.2.6/reference/#connecting-via-java)
+client. Sumbit scripts as a `String` for evaluation on the server.
+* [Remote Graph](http://tinkerpop.apache.org/docs/3.2.6/reference/#connecting-via-remotegraph)
+uses the same client connection but enables you to compose queries as code.
+
+## JanusGraph configuration
+
+### Gremlin Server configuration
+
+* Select a specific storage backend to use, then copy its configuration into
+the `$JANUSGRAPH_HOME/conf/gremlin-server/` directory
```

Comment:
Is this still in the context of a source JanusGraph repository or is it a binary distribution? If source, same comment as above regarding the presence of `conf` in root directory here and below. If binary, make a note that a separate binary distribution is assumed for managing the Gremlin Server instance.

Reply:
The context for all the examples is the binary distribution. The examples are packaged with the distribution so users can download the distribution then build the examples.